internal/mock/sqladmin.go

```diff
@@ -148,12 +160,20 @@ func InstanceGetSuccess(i FakeCSQLInstance, ct int) *Request {
 	return r
 }
 
-// CreateEphemeralSuccess returns a Request that responds to the
-// `sslCerts.createEphemeral` SQL Admin endpoint. It responds with a "StatusOK" and a
-// SslCerts object.
+// InstanceGetSuccess returns a Request that responds to the `instance.get` SQL Admin
+// endpoint. It responds with a "StatusOK" and a DatabaseInstance object.
 //
-// https://cloud.google.com/sql/docs/mysql/admin-api/rest/v1beta4/sslCerts/createEphemeral
-func CreateEphemeralSuccess(i FakeCSQLInstance, ct int) *Request {
+// https://cloud.google.com/sql/docs/mysql/admin-api/rest/v1beta4/instances/get
+//
+// To Configure the response from the SQL Admin endpoint, use
+// InstanceGetSuccessWithOpts.
+func InstanceGetSuccess(i FakeCSQLInstance, ct int) *Request {
+	return InstanceGetSuccessWithDatabase(i, ct, &sqladmin.DatabaseInstance{})
```

Comment:
I don't really think relying on the `&sqladmin.DatabaseInstance{}` struct is a good idea - it's a part of an API not under out control, and overlaps with FakeCSQLInstance. I think we should really be treating "FakeCSQLInstance" as  an options holder. If we need to change something in the SQLAdmin.DatabaseInstance{} struct, let's do it by changing FakeCSQLInstance.